.travis.yml

```diff
@@ -1,16 +1,20 @@
-language: objective-c # lol
-osx_image: xcode9.2
+language: objective-c
+osx_image: xcode9.3
 xcode_project: DVR.xcodeproj
 
-script: xcodebuild -scheme "$TRAVIS_XCODE_SCHEME" -sdk "$TRAVIS_XCODE_SDK" -destination "$DESTINATION" test
-
 matrix:
   include:
     - xcode_scheme: DVR-iOS
       xcode_sdk: iphonesimulator
       env:
-        - DESTINATION="OS=10.1,name=iPhone 7 Plus"
+        - DESTINATION="iOS Simulator,OS=11.3,name=iPhone X"
     - xcode_scheme: DVR-macOS
       xcode_sdk: macosx
       env:
-        - DESTINATION="arch=x86_64"
```

Comment:
Did this need to change? I think we want to prefer attribute-based selection of destinations, rather than names (which Travis could change from underneath us at any time).

Reply:
I can try reverting this change and see how it goes 👍

Reply:
so it turns out that both the platform and the architecture must be specified. the final command (with env variables filled in) is:

```bash
xcodebuild -jobs `sysctl -n hw.ncpu` test -project DVR.xcodeproj -scheme DVR-macOS -sdk macosx \
  -destination "platform=macOS,arch=x86_64" ONLY_ACTIVE_ARCH=YES CODE_SIGNING_IDENTITY="" CODE_SIGNING_REQUIRED=NO | xcpretty -c
```

I'm hoping Apple doesn't change the operating system name that often :-D

Reply:
Awesome! I think `"platform=macOS,arch=x86_64"` is much better than `OS X`.

(especially when we need to do `"platform=macOS,arch=arm64"` down the line 😉 )